products/nvidiadriver.md

```diff
@@ -70,17 +84,17 @@ releases:
 -   releaseCycle: "R470-Windows"
     lts: true
     support: 2021-09-20
-    eol: 2024-09-01
+    eol: 2024-07-01
     latest: "474.04"
     link: https://www.nvidia.com/Download/driverResults.aspx/196634/
     releaseDate: 2021-06-22
     
 -   releaseCycle: "R470-Linux"
     lts: true
     support: 2021-10-26
-    eol: 2024-09-01
-    latest: "470.161.03"
-    link: http://www.nvidia.com/Download/driverResults.aspx/194637/
+    eol: 2024-07-01
```

Comment:
@marcwrobel I sourced that data from [NVIDIA Documentation](https://docs.nvidia.com/datacenter/tesla/drivers) in Table 2.

I'll at least update the date to be three years from the release date. I'm sure we'll get an update as the time gets closer if it changes. (The releases.json file also doesn't appear to be up to date yet.)

Reply:
I think it's fine to update it to the date NVIDIA states in that doc for the EOL date (even though it technically conflicts with other info in the same doc), and update it as we get closer to the date if they do. 

> (The releases.json file also doesn't appear to be up to date yet.)

I believe it isn't because they don't include all branches in that, just the ones they deem production ready